static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -143,7 +136,18 @@ const showBanner = (params: EngagementBannerParams): void => {
     }).show(renderedBanner);
 
     if (messageShown) {
-        recordInteraction(params.interactionOnMessageShow);
+        submitInsertEvent(
+            'ACQUISITIONS_ENGAGEMENT_BANNER',
+            params.products,
+            params.campaignCode
+        );
+
+        // TODO: QUESTION: for things which are automatically in view on insert, should we fire both events?
```

Comment:
@jranks123 @Mullefa @desbo @dominickendrick... any opinions on this? My instinct is yes we should as it makes the data more consistent